bgp/bgp_api_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017 Pantheon technologies s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bgp
+
+import (
+	"github.com/ligato/cn-infra/logging/logrus"
+	"github.com/stretchr/testify/assert"
+	"net"
+	"reflect"
+	"testing"
+)
+
+func TestToChan(t *testing.T) {
```

Comment:
Please explain in comment what is the intent of this test

Reply:
comment added